db/schema.sql

```diff
@@ -63,8 +63,22 @@ CREATE TABLE IF NOT EXISTS team_student_weekly_contributions (
   num_commits integer NOT NULL
 );
 
-
-
-
-
+CREATE TABLE IF NOT EXISTS student_attendance (
+    id SERIAL PRIMARY KEY,
+    enrollment_id integer REFERENCES students ON DELETE CASCADE,
```

Comment:
We'll need to revise enrollment_id once the schema is modified.  Also, be careful about cascading deletes.  We don't always want to delete historic data from our systems.  In particular, if a student is removed from the program, it's important to maintain a history of their attendance.

This will be a moot point since separations will strictly be a change in status but it's good to be certain that cascading deletes are encoded into the table constraints.

Please remove the cascading delete constraint for now.